src/sensor.cpp

```diff
@@ -1082,4 +1082,76 @@ namespace librealsense
     {
         register_metadata(RS2_FRAME_METADATA_BACKEND_TIMESTAMP,     make_additional_data_parser(&frame_additional_data::backend_timestamp));
     }
+
+    iio_hid_timestamp_reader::iio_hid_timestamp_reader()
+    {
+        counter.resize(sensors);
+        reset();
+    }
+
+    void iio_hid_timestamp_reader::reset()
+    {
+        std::lock_guard<std::recursive_mutex> lock(_mtx);
+        started = false;
+        for (auto i = 0; i < sensors; ++i)
+        {
+            counter[i] = 0;
+        }
+    }
+
+    rs2_time_t iio_hid_timestamp_reader::get_frame_timestamp(const request_mapping& mode, const platform::frame_object& fo)
```

Comment:
This function has recently been updated in https://github.com/IntelRealSense/librealsense/pull/3726
Please review and merge the changes manually